styles/style.css

```diff
@@ -1,387 +1,305 @@
+  padding: var(--padding-big) 0;
+}
 
-  .image-container-cars div {
-    width: 100%;
-  }
+.image-container-cars img {
+  padding: 0.5rem;
+  border: 2px solid white;
+  object-fit: contain;
+  height: 50vh;
+  width: auto;
 }
 
-/* Contact section */
+/*  ----------------Contact Section---------------- */
 .contact-section {
   background-image: url(/images/swansway-motor-group-wnz9awGVA9w-unsplash.jpg);
   background-size: cover;
   background-repeat: no-repeat;
+  background-position: center;
   text-align: center;
-  height: 100vh;
+  height: var(--dvh-with-fallback);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
 }
 
-.heading-contact h1 {
-  letter-spacing: 0.2rem;
-  padding-left: 0.6rem;
-  padding-right: 0.6;
-  padding-top: 5rem;
-  margin-bottom: 2.6rem;
-  font-weight: 800;
+.contact-section h1 {
+  font-weight: var(--font-weight-heading);
+  font-size: var(--font-size-heading);
 }
 
-.paragraph-contact p {
-  font-size: 1.3rem;
-  margin-top: 0.8rem;
-  margin-bottom: 3rem;
-  padding: 0.5rem;
+.contact-section p {
+  font-weight: 500;
+  font-size: var(--font-size-paragraph);
 }
 
 .forms {
   display: flex;
+  flex-direction: column;
+  align-items: center;
   justify-content: center;
-  padding: 1rem;
-  margin: 1rem;
-  background-color: hsla(105, 6%, 13%, 0.466);
-  max-width: 1050px;
+  padding: var(--padding);
+  background-color: hsla(0, 0%, 0%, 0.616);
+  max-width: max-content;
+  margin-top: var(--padding);
+  gap: 1rem;
 }
 
 .input-group {
-  padding: 0.8rem;
+  display: flex;
+  flex-direction: column;
+  width: 35vw;
+  gap: 0.5rem
 }
 
-input,
-textarea {
-  width: 90%;
-  max-width: 1050px;
+.forms input,
+.forms textarea {
+  resize: none;
   padding: 0.5rem;
-  outline: 0;
   border: 1px solid rgb(240, 240, 240);
   background: hsla(99, 7%, 13%, 0.4);
   cursor: text;
   color: white;
 }
 
-button {
-  padding: 0.4rem;
-  width: 60%;
-  border-radius: 7px;
+.forms label {
+  font-size: 1.4rem;
+}
+
+.forms button {
+  width: 100%;
+  padding: 0.6rem;
+  font-size: 1rem;
+  font-weight: 700;
 }
 
-button:active {
+.forms button:active {
   background-color: transparent;
   color: white;
 }
 
-/* tablet */
-@media screen and (min-width: 706px) {
-  .forms {
-    width: 550px;
-    margin: auto;
-  }
-}
-@media screen and (min-width: 768px) and (orientation: portrait) {
-  .heading-contact h1 {
-    font-size: 4rem;
-    padding: 3rem;
-  }
 
-  .paragraph-contact p {
-    padding: 3rem;
-    font-size: 1.5rem;
-  }
-  .forms form {
-    padding: 2rem;
-  }
-}
+@media only screen and (max-width: 800px) {
 
-@media screen and (min-width: 768px) and (orientation: landscape) {
-  .contact-section {
-    background-position: center;
-  }
-  .heading-contact h1 {
-    font-size: 4rem;
-  }
-  .forms {
-    width: 60%;
-    margin: auto;
+  /*  ----------------Landing Section---------------- */
+  .landing-section {
+    justify-content: center;
+    gap: 2rem
```

Comment:
so the way i approach it is - copy the whole .lading-section class:
```
.landing-section {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  height: var(--dvh-with-fallback);
  width: 100%;
  padding: var(--padding);
  background-image: url(/images/balkouras-nicos--tJcdoXAtlQ-unsplash.jpg);
  background-position: center;
  background-size: cover;
}
```

and modify only the things you need to modify:
```
.landing-section {
  display: flex;
  flex-direction: column;
  **justify-content: center;
  gap: 2rem;**
  height: var(--dvh-with-fallback);
  width: 100%;
  padding: var(--padding);
  background-image: url(/images/balkouras-nicos--tJcdoXAtlQ-unsplash.jpg);
  background-position: center;
  background-size: cover;
}
```
So now you've overwritten two properties. The rest is the same and you don't need it here, as it's already present and we don't need duplicate code. So you remove it. 

So now it's super clear what is changing when you hit the media query.